schemas/com.snowplowanalytics.snowplow.ecommerce/refund/jsonschema/1-0-0

```diff
@@ -0,0 +1,43 @@
+{
+    "$schema": "http://iglucentral.com/schemas/com.snowplowanalytics.self-desc/schema/jsonschema/1-0-0#",
+    "description": "Schema for a refund in Ecommerce",
+    "self": {
+        "vendor": "com.snowplowanalytics.snowplow.ecommerce",
+        "name": "refund",
+        "format": "jsonschema",
+        "version": "1-0-0"
+    },
+    "type": "object",
+    "properties": {
+        "transaction_id": {
+            "description": "The ID of the transaction.",
```

Comment:
Nit: knowing nothing about ecommerce, I am just wondering if it's safe to assume that we also have the ID of the refunded transaction available or should we rather make it optional?